app/models/manageiq/providers/openshift/inventory/parser/infra_manager/full_refresh.rb

```diff
@@ -0,0 +1,20 @@
+class ManageIQ::Providers::Openshift::Inventory::Parser::InfraManager::FullRefresh < ManageIQ::Providers::Kubevirt::Inventory::Parser::FullRefresh
+
+  private
+
+  def provider_class
+    ManageIQ::Providers::Openshift
+  end
+
+  def product
+    "OpenShift Virtualization"
+  end
+
+  def vendor
+    "openshift_infra"
+  end
+
+  def version
+    '0.1.0'
+  end
```

Comment:
These will probably need to be added to the `::PartialRefresh` and `::PartialTargetRefresh` classes since they use the common kubevirt parser class https://github.com/ManageIQ/manageiq-providers-kubevirt/pull/247/files#diff-ad8b5bb422b15e9a6a52b7f4ded177f8bce041245f2a05d5dd983a09f6e055c6R68-R72

@agrare Should we move these to a common mixin and include to the classes that need it? Kind of like https://github.com/ManageIQ/manageiq-providers-kubevirt/blob/master/app/models/manageiq/providers/kubevirt/constants.rb, or open to any ideas you may have

Reply:
Since these are static values we could move them to the persister definition as a default value.

Reply:
For example ovirt does a similar trick to work with the RHV subclass: https://github.com/ManageIQ/manageiq-providers-ovirt/blob/master/app/models/manageiq/providers/ovirt/inventory/persister/infra_manager.rb#L9-L11